service/src/test/java/uk/gov/hmcts/reform/fpl/service/payment/PaymentServiceTest.java

```diff
@@ -99,6 +104,48 @@ void setup() {
             when(feeService.getFeesDataForC2(WITHOUT_NOTICE)).thenReturn(buildFeesData(feeForC2WithoutNotice));
         }
 
+        @Test
+        void shouldRetryPaymentsApiWhen500IsThrown() {
+            CreditAccountPaymentRequest expectedPaymentRequest = testCreditAccountPaymentRequestBuilder()
+                .customerReference("1")
+                .amount(feeForC2WithNotice.getCalculatedAmount())
+                .fees(List.of(feeForC2WithNotice))
+                .build();
+
+            Mockito.doThrow(FeignException.InternalServerError.class).when(paymentApi).createCreditAccountPayment(
+                any(), any(), any());
+
+            try {
```

Comment:
we are using assertThrows
e.g
            UserLookupException actualException = assertThrows(UserLookupException.class,
                () -> organisationService.findUserByEmail(email));

Reply:
done